src/mesh/Draco_Mesh.cc

```diff
@@ -274,6 +286,26 @@ void Draco_Mesh::compute_cell_to_cell_linkage(
         // populated cell-to-ghost-cell linkage
         cell_to_ghost_cell_linkage[cell].push_back(
             std::make_pair(ghost_cells_in_common[0], vec_node_vec[l]));
+
+        // set bool to indicate a condition for face l has been found
+        has_face_cond = true;
+      }
+
+      // make face a boundary if no face conditions have been found
+      if (!has_face_cond) {
+
+        // augment side flags with vacuum b.c.
```

Comment:
The unit tests don't cover this block of code (`!has_face_cond`).  How difficult would it be to add a test that would use this code block?

Reply:
@KineticTheory This won't be difficult at all - simply a matter of not loading boundary data for one of the test meshes.  It will be added shortly.

Reply:
@KineticTheory The unit test `tstDraco_Mesh` has been augmented to test this block of code - apologies for having added this sooner.